clustering.Rmd

```diff
@@ -45,95 +49,147 @@ use the subgroups to generate new questions about the data and follow up with a
+or values to help us. 
+This approach has both advantages and disadvantages. 
+Clustering requires no additional annotation or input on the data. 
+For example, it would be nearly impossible to annotate 
+all the articles on Wikipedia with human-made topic labels. 
+However, we can still cluster the articles without this information 
+to find groupings corresponding to topics automatically. 
+
+However, because there is no response variable, it is not as easy to evaluate
 the "quality" of a clustering.  With classification, we can use a test data set
 to assess prediction performance. In clustering, there is not a single good
 choice for evaluation. In this book, we will use visualization to ascertain the
 quality of a clustering, and leave rigorous evaluation for more advanced
 courses.  
 
-> There are also so-called *semisupervised* tasks, \index{semisupervised} where only some of the data
-> come with labels/annotations, but the vast majority don't. The goal
-> is to try to uncover underlying structure in the data that allows one to
-> guess the missing labels. This sort of task is beneficial, for example, when one 
-> has an unlabelled data set that is too large to manually label, but one is willing to
-> provide a few informative example labels as a "seed" to guess the labels for all the data.
-
+As in the case of classification, 
+there are many possible methods that we could use to cluster our observations 
+to look for subgroups. 
+In this book, we will focus on the widely used K-means \index{K-means} algorithm.
+In your future studies, you might encounter hierarchical clustering,
+principle component analysis, multidimensional scaling, and more; 
+see the additional resources section at the end of the next chapter 
+for where to begin learning more about these other methods.
+
+> There are also so-called *semisupervised* tasks, \index{semisupervised} 
+> where only some of the data come with response variable labels/values, 
+> but the vast majority don't. 
+> The goal is to try to uncover underlying structure in the data 
+> that allows one to guess the missing labels. 
+> This sort of task is beneficial, for example, 
+> when onehas an unlabelled data set that is too large to manually label, 
+> but one is willing to provide a few informative example labels as a "seed" 
+> to guess the labels for all the data.
 
 **An illustrative example** 
 
-
 Here we will present an illustrative example using a data set \index{Palmer penguins} from the
 [{palmerpenguins} R data package](https://allisonhorst.github.io/palmerpenguins/). This data set was
 collected by [Dr. Kristen Gorman](https://www.uaf.edu/cfos/people/faculty/detail/kristen-gorman.php) and
-the [Palmer Station, Antarctica LTER](https://lternet.edu/) and includes
-measurements for adult penguins near Palmer Station [@palmerpenguins]. We have
+the [Palmer Station, Antarctica Long Term Ecological Research Site](https://pal.lternet.edu/) and includes
```

Comment:
this is confusing wording

"this data set was collected by [person] and [station, site] and includes msrmts..."

Reply:
I know... but that is what the source says... The station is literally listed as an author here: https://allisonhorst.github.io/palmerpenguins/#references